tests/tck/features/match/Base.IntVid.feature

```diff
@@ -137,7 +137,7 @@ Feature: Basic match
       | "serve" | "Cavaliers" |
     When executing query:
       """
-      MATCH (v1:player{name: "LeBron James"}) -[r:serve]-> (v2 {name: "Cavaliers"})
+      MATCH (v1:player{name: "LeBron James"}) -[r:serve]-> (v2:team{name: "Cavaliers"})
```

Comment:
FYI. Since all property names must be attached to a tag name, isn't this more intuitive to design the syntax:
```
MATCH (v1:player{player.name: "LeBron James"}) -[r:serve]-> (v2{team.name: "Cavaliers"})
```